File: src/main/java/com/patres/timetable/security/AuthoritiesConstants.java
package com.patres.timetable.security;

/**
 * Constants for Spring Security authorities.
 */
public final class AuthoritiesConstants {

    public static final String ADMIN = "ROLE_ADMIN";

    public static final String SCHOOL_ADMIN = "ROLE_SCHOOL_ADMIN";

    public static final String ANONYMOUS = "ROLE_ANONYMOUS";

    public static final String TEACHER = "ROLE_TEACHER";

    public static final String TIMETABLE_WATCHER = "ROLE_TIMETABLE_WATCHER";


    private AuthoritiesConstants() {
    }
}

File: src/main/java/com/patres/timetable/security/AuthoritiesConstants.kt
package com.patres.timetable.security

/**
 * Constants for Spring Security authorities.
 */
object AuthoritiesConstants {

    const val ADMIN = "ROLE_ADMIN"
    const val SCHOOL_ADMIN = "ROLE_SCHOOL_ADMIN"
    const val ANONYMOUS = "ROLE_ANONYMOUS"
    const val TEACHER = "ROLE_TEACHER"
    const val TIMETABLE_WATCHER = "ROLE_TIMETABLE_WATCHER"
}
